ext/src/ruby_api/store.rs

```diff
@@ -163,40 +163,28 @@ impl Store {
     }
 
     /// @yard
-    /// Returns the amount of fuel consumed by this {Store}’s execution so far,
-    /// or +nil+ when the {Engine}’s config does not have fuel enabled.
-    /// @return [Integer, Nil]
-    pub fn fuel_consumed(&self) -> Option<u64> {
-        self.inner_ref().fuel_consumed()
+    /// Returns the amount of fuel in the {Store}.
+    /// This function errors if fuel consumption is not configured by passing
+    /// enabling fuel consumption via Wasmtime::Engine::new.
+    ///
+    /// @return [Integer, Error]
```

Comment:
The exception isn't returned but raised, we should probably leave it as `@return [Integer]`

Reply:
I think that what I need is a `@raise`. I want to signal that this method raises. I've added that here and below.